src/js/tabs/osd.js

```diff
@@ -1014,14 +1022,20 @@ OSD.chooseFields = function () {
                   OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
                     F.ANTI_GRAVITY
                   ]);
-                  if (semver.gte(CONFIG.apiVersion, "1.40.0")) {
-                    OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
-                        F.G_FORCE
-                    ]);
+                  if (semver.gte(CONFIG.apiVersion, "1.39.0")) {
+                      OSD.constants.DISPLAY_FIELDS = OSD.constants.DISPLAY_FIELDS.concat([
+                        F.ANTI_GRAVITY
```

Comment:
Something went wrong in te rebase, the version 1.39 with ANTI_GRAVITY element appears twice.

Reply:
Okay, I removed the duplicate and changed the order of the elements to match the ordering at other places.